src/js/app.js

```diff
@@ -57,6 +59,18 @@ var app = {
         $('#modalDelete-link').attr('href', $(this).data('link'));
         $('#modalDelete').modal('show');
     },
+    modalVanity: function () {
+        var id = $(this).data('id');
+        $('#modalVanity').modal('show');
+        $('#modalVanity-link').click(function () {
+            var vanity = $('#modalVanity-input').val();
+            var $callerButton = $(this);
+            $.post(window.AppConfig.base_url + '/upload/' + id + '/vanity', {vanity: vanity}, function () {
+                $callerButton.tooltip('dispose');
+                window.location.href = window.AppConfig.base_url + '/home';
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏the issue here is if you are updating a media deep in the pagination or in the public page, you are going to be redirected to the home page and than go back where you were before. I would update the HTML using the data returned by the post request instead of performing a redirect.

Reply:
Sorry, must of missed that before I made the PR. I had the redirect thing as a test to make sure that everything was working properly, and was going to do the frontend stuff once it was. Again, sorry for missing that detail. Fixed :+1: